process/modules/converb_to_conj.py

```diff
@@ -0,0 +1,180 @@
+from process.module import ParaphraseModule
+from process.preprocessing_utils import PreprocessingUtils
+from stanza.utils.conll import CoNLL
+
+import stanza
+import pymorphy2 
+morph = pymorphy2.MorphAnalyzer()
+import pyconll
+import udon2
+
+class ConverbToConjuctionModule(ParaphraseModule):
+    def __init__(self, name="capitalize_nouns") -> None:
+        super().__init__(name=name)
+    
+    def load(self, preproc_utils: PreprocessingUtils) -> None:
+        # load any tools as `preproc_utils` attributes
```

Comment:
MorphAnalyzer можно сделать атрибутом preproc_utils, как станзу:

```suggestion
        if getattr(preproc_utils, 'morph', None) is None:
            preproc_utils.morph = pymorphy2.MorphAnalyzer()
```

Тогда придется передавать его как аргумент в функции, обращающиеся к `morph`. См. на примере функции `chose_converb`.